.openshift-ci/jobs/integration-tests/env.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 CI_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
-# shellcheck source=SCRIPTDIR=../../scripts/lib.sh
+# shellcheck source=SCRIPTDIR/../../scripts/lib.sh
```

Comment:
Why is the = removed here.

Reply:
it was invalid for the spellcheck directive, because it's expecting a path. `SCRIPTDIR` is a constant that shellcheck treats as the directory containing the script it's currently processing, so any source directives need to be relative to that directory

Reply:
I didn't look closely. I did not notice that there were two equal signs or that this was attempting to set SCRIPTDIR equal to something.